mne_bids_pipeline/scripts/init/_00_init_derivatives_dir.py

```diff
@@ -49,7 +47,7 @@ def init_subject_dirs(
     *,
     cfg,
     subject: str,
-    session: Optional[str] = None
+    session: str,
```

Comment:
@larsoner Why did you remove the `Optional` type in many places?

Maybe there's a (common) misunderstanding here: `Optional[foo]` is actually shorthand for `Union[None, foo]`, which can (finally!!) be written as `None | foo` in Python 3.10 (this was sooo overdue).

You can, therefore, have a **required** parameter that is nevertheless annotated with an `Optional` type.

Reply:
Ahh yes whatever can be None can be restored then. Or maybe we should just wait until 3.10 is the minimum requirement and use the nicer syntax. Up to you